MPF.Processors/Redumper.cs

```diff
@@ -546,6 +578,50 @@ internal override List<OutputFile> GetOutputFiles(string? baseDirectory, string
         private static bool DatfileExists(string log)
             => GetDatfile(log) != null;
 
+        /// <summary>
+        /// Copies a file with the header removed
+        /// </summary>
+        /// <param name="inputFilename">Filename of file to copy from</param>
+        /// <param name="outputFilename">Filename of file to copy to</param>
+        /// <param name="headerLength">Length of header to remove</param>
+        private static bool RemoveHeader(string inputFilename, string outputFilename, int headerLength = 4)
+        {
+            // If the file doesn't exist, we can't copy
+            if (!File.Exists(inputFilename))
+                return false;
+            
+            // If the output file already exists, don't overwrite
+            if (File.Exists(outputFilename))
+                return false;
+            
+            try
+            {
+                using var inputStream = new FileStream(inputFilename, FileMode.Open, FileAccess.Read);
+                
+                // If the header length is not valid, don't copy
+                if (headerLength < 1 || headerLength >= inputStream.Length)
+                    return false;
+
+                using var outputStream = new FileStream(outputFilename, FileMode.Create, FileAccess.Write);
+
+                // Skip the header
+                inputStream.Seek(headerLength, SeekOrigin.Begin);
+
+                // inputStream.CopyTo(outputStream);
+                byte[] buffer = new byte[4096];
+                int count;
+                while ((count = inputStream.Read(buffer, 0, buffer.Length)) != 0)
+                    outputStream.Write(buffer, 0, count);
```

Comment:
Please add `{ }` around this statement. It follows the same rules as `for` loops in this codebase.